Manifest.toml

```diff
@@ -0,0 +1,55 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
I don't think you wanted to include the Manifest file here.

Reply:
yeah I don't know why that's there - it's not in the .gitignore file?

Reply:
Probably not cause this package pre-dates Julia 1.0 and Manifest.toml files.